Detectors/TRD/simulation/src/Digitizer.cxx

```diff
@@ -165,9 +166,6 @@ bool Digitizer::convertHits(const int det, const std::vector<HitType>& hits, TRD
   const int nRowMax = padPlane->getNrows();
```

Comment:
I don't see where you update the padPlane. This will have to be done for every detector, because there are different numbers of pad rows.

Reply:
Two lines above: `const TRDPadPlane* padPlane = mGeo->getPadPlane(det);`.